### ISSUE TITLE
Albert's Task (10/2/23-10/13/23)

### ISSUE THREAD

**Issue by liluzibird** [natt96z/cybersac#120]
- Media department testimonials (WIP/Pending)
- Still Try to Reach MSA-Santa Ana 
- Zoom Meeting Thursday 3:30PM 
- Spell Check - Review Student Feedback (CyberSAC Material)
- Improve Current and Future Material (OneDrive Material)
- Work on Script code
-----------------------------------------------------------------------------------------------------------------
Still make phone calls if there is a possible extension to additional schools. We have a meeting this Thursday on Zoom at 3:30, we are going to possibly discuss team virtual machine and school scheduling.

**Comment by natt96z** [comment]
![image](https://github.com/natt96z/cybersac/assets/112590005/5ac5edef-4f04-486d-a75c-5e8479a2e783)

**Monday October 2nd, 2023** - Me and @natt96z  visited Magnolia Science Academy to talk to Mr. K, Mr Quarez and the computer department, we were given a tour of the school, I CC'd all completed work to the appropriate party. We were then shown the schools computer hardware and later led into the Principals conference room to complete registration.

**Comment by natt96z** [comment]
![image](https://github.com/natt96z/cybersac/assets/112590005/ff3230ce-9fe7-4c23-adf2-02a7f6a628f1)

**Wednesday October 4th 2023** - Code Cleanup, sites themes configuration was updated again to fix loading bucks and compile issues. Website version is now 5.0.1. With help from team mate @natt96z  we had to add a new _themes directory to github and update the api theming engine to the latest RTD version for strange security reasons.

**Comment by natt96z** [comment]
![Screenshot from 2023-10-05 16-55-43](https://github.com/natt96z/cybersac/assets/112590005/d9354404-31bc-45bf-a879-7cac34ca436a)

**Thursday October 5th** - Zoom Meeting with group, we discussed the MSA visit, future school prospects and adding quality of life improvements to the team website. There is a possible issue with MSA enrollment thats in progress of being fixed.

**Comment by natt96z** [comment]
![image](https://github.com/natt96z/cybersac/assets/112590005/6b2d949c-51d3-4db6-b51a-066bf01518fa)
![image](https://github.com/natt96z/cybersac/assets/112590005/69fcb0d7-e890-4354-8669-64a14bf08e3c)


**10/11/2023 - Me, Patrice and Nate** reviewed Alicias OneDrive that was sent over the school email, as a team we also made sure content was relatable to what we already had on Cybersac and PowerPoint material improvements as well. Another followup email was sent to MSA with Albert's help in the process as well emailing them. I personally reorganized our original OneDrive as well. Nothing from Alicia's drive was added to ours.
